src/test/lib/storage/reference_segment_test.cpp

```diff
@@ -36,8 +36,11 @@ class ReferenceSegmentTest : public BaseTest {
     TableColumnDefinitions column_definitions2;
     column_definitions2.emplace_back("a", DataType::Int, false);
     column_definitions2.emplace_back("b", DataType::Int, false);
-    _test_table_dict = std::make_shared<opossum::Table>(column_definitions2, TableType::Data, 5, UseMvcc::Yes);
-    for (int i = 0; i <= 24; i += 2) _test_table_dict->append({i, 100 + i});
+    _test_table_dict =
+        std::make_shared<opossum::Table>(column_definitions2, TableType::Data, ChunkOffset{5}, UseMvcc::Yes);
+    for (int i = 0; i <= 24; i += 2) {
```

Comment:
Since you touch this line:
- rename `i`
- be explicit with the data type: `uint8_t`

Reply:
Right - I should have checked the loop body in more detail :) But we can still be specific and use `int32_t`.

Reply:
Ah yes, right.